### PR TITLE
Modified repl.py to conform to follow more PEP standards including 8 and 263

### PR DESCRIPTION
**Pull request by shanahanjrs**
This merge includes style and indentation fixes, removes unused function args, and updates variable names to be more descriptive and improve code readability.

### CODE REVIEW

File: bpython/repl.py [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+"""
```
Comment (sebastinas) [inline bpython/repl.py]: Why? We consistently use #-commented license headers, so please do not change that.

File: bpython/repl.py [hunk 2]
```diff
@@ -1099,15 +1103,15 @@ def open_in_external_editor(self, filename):
...
+        if not os.path.isfile(self.config.config_path):
             if self.interact.confirm(_("Config file does not exist - create "
                                        "new from default? (y/N)")):
                 try:
                     default_config = pkgutil.get_data('bpython',
                                                       'sample-config')
                     if py3:  # py3 files need unicode
                         default_config = default_config.decode('ascii')
-                    bpython_dir, script_name = os.path.split(__file__)
+                    #bpython_dir, script_name = os.path.split(__file__)
```
Comment (sebastinas) [inline bpython/repl.py]: Just remove it if `bpython_dir` and `script_name` are unused.

File: bpython/repl.py [hunk 3]
```diff
@@ -56,6 +58,7 @@
 
 
 class RuntimeTimer(object):
+    """Calculate running time"""
```
Comment (thomasballinger) [inline bpython/repl.py]: I like docstrings, maybe we can do one better:
`"""Records cumulative time spend in this context manager"""`

File: bpython/repl.py [hunk 4]
```diff
@@ -601,7 +604,7 @@ def get_args(self):
                 try:
                     fake_cursor = self.current_line.index(func) + len(func)
                     f = simpleeval.evaluate_current_attribute(
-                            fake_cursor, self.current_line, self.interp.locals)
+                        fake_cursor, self.current_line, self.interp.locals)
```
Comment (thomasballinger) [inline bpython/repl.py]: I'm not very familiar with these rules, is this recommended by PEP8? I interpret https://www.python.org/dev/peps/pep-0008/#indentation as saying the original is correct.

File: bpython/repl.py [hunk 5]
```diff
@@ -770,13 +773,13 @@ def line_is_empty(line):
             indentation = 0
         return indentation
 
-    def formatforfile(self, s):
+    def formatforfile(self, stdout):
```
Comment (thomasballinger) [inline bpython/repl.py]: I agree `s` isn't great, but at least there's the convention that `s` is a string. Stdout sounds like a file descriptor or sys.stdout or something to me. How about `session_output`?

File: bpython/repl.py [hunk 6]
```diff
@@ -817,11 +820,11 @@ def write2file(self):
                 self.interact.notify(_('Save cancelled.'))
                 return
 
-        s = self.formatforfile(self.getstdout())
+        stdout_text = self.formatforfile(self.getstdout())
```
Comment (thomasballinger) [inline bpython/repl.py]: This isn't just stdout text, it's session text; the interleaving of stdin, stdout, and stderr.

File: bpython/repl.py (hunk 3 above)
Comment (thomasballinger) [inline bpython/repl.py]: Or `Tracks cumulative time spend in this context`

File: bpython/repl.py [hunk 7]
```diff
@@ -1069,8 +1072,9 @@ def tokenize(self, s, newline=False):
     def clear_current_line(self):
         """This is used as the exception callback for the Interpreter instance.
         It prevents autoindentation from occurring after a traceback."""
+        # XXX: Empty function
```
Comment (thomasballinger) [inline bpython/repl.py]: Is this a PEP or a pylint thing? Why is this useful?

File: bpython/repl.py (hunk 3 above)
Comment (shanahanjrs) [inline bpython/repl.py]: Sure those sound much better! Ill make the changes you purposed and resubmit my pull request.

File: bpython/repl.py (hunk 1 above)
Comment (Trundle) [inline bpython/repl.py]: It would also change the license header from a comment to the docstring of the `bpython.repl` module.

**Comment by sebastinas** [comment]
Squashed the two commits and rebased them on current master. Thanks.

**Pull request closed without merging**